apis/management.cattle.io/v3/authz_types.go

```diff
@@ -45,10 +45,9 @@ type ProjectCondition struct {
 }
 
 type ProjectSpec struct {
-	DisplayName                   string `json:"displayName,omitempty" norman:"required"`
-	Description                   string `json:"description"`
-	ClusterName                   string `json:"clusterName,omitempty" norman:"required,type=reference[cluster]"`
-	PodSecurityPolicyTemplateName string `json:"podSecurityPolicyTemplateName,omitempty" norman:"type=reference[podSecurityPolicyTemplate]"`
+	DisplayName string `json:"displayName,omitempty" norman:"required"`
+	Description string `json:"description"`
+	ClusterName string `json:"clusterName,omitempty" norman:"required,type=reference[cluster]"`
```

Comment:
where did the pspt field go? you should be adding that to projectStatus, correct?

Reply:
Added status to: https://github.com/rancher/types/pull/294/files#diff-2e11c852ab2dcd25c29b83183a648fa3R30